arviz/stats/diagnostics.py

```diff
@@ -657,20 +637,22 @@ def _ess(ary):
     ess = ess / tau_hat
     if np.isnan(rho_hat_t).any():
         ess = np.nan
+    if relative:
+        ess /= ary.size
```

Comment:
I was thinking on `ess = 1 if relative else n_chain * n_draw` some lines before instead of `ess = n_chain * n_draw`. What do you think about it?

Reply:
Yes, this makes sense

Reply:
I did something close to this. (1/log10(1)) is not good :)